packages/client-twitter/src/interactions.ts

```diff
@@ -282,6 +290,37 @@ export class TwitterInteractionClient {
             this.client.saveRequestMessage(message, state);
         }
 
+        const pizzaCheck = `
+        You are checking to see if someone is asking you to order a pizza.
+        They should explicitly ask for a pizza order.
+
+        Here is the tweet they posted:
+        ${currentPost}`
+        + pizzaDecisionFooter;
+
+        const pizzaCheckResponse = await generateText({
+            runtime: this.runtime,
+            context: pizzaCheck,
+            modelClass: ModelClass.LARGE,
+        });
+
+        console.log("[PIZZA-GEN][INTERACTIONS CLIENT] PIZZA check response: ", pizzaCheckResponse, " ", currentPost);
```

Comment:
Thanks for CRUSHING this you are the GOAT for doing this! Can we elizaLogger everywhere instead